community/modules/compute/gke-node-pool/variables.tf

```diff
@@ -176,6 +176,15 @@ variable "labels" {
   type        = map(string)
 }
 
+variable "node_labels" {
```

Comment:
My only thought on this is that there could be some user confusion between `labels` and `node_labels`. Due to HPC Toolkit mechanics around resource labels, the `labels` variable name cannot be changed.

Since we have already strayed from the names used by the terraform resource, I am thinking that `kubernetes_labels` or `kubernetes_node_labels` would be a better name as it would make the distinction clearer. I am leaning toward the prior and dropping the word `node` from the name as it possibility seems redundant since this is part of the `node-pool` module.

WDYT?

Reply:
My first instinct was to use `kubernetes_labels` but I second guessed myself, worrying it was too long as a variable name, so it was highly amusing for me to get this feedback. 😂

FWIW, for me, naming is hard because the idea is for the framework to be opinionated and therefore not just mirror the terraform modules, yet on the other hand when users need to customize, it's much easier for the engineers involved to have similar (or identical) names.

I'll make the change.